docs/_automation/powershell/README.md

```diff
@@ -34,10 +34,22 @@ The FinOps toolkit PowerShell module is a collection of commands to automate and
 
 ## 📥 Install the module
 
+The FinOps toolkit module requires PowerShell 7, which is supported on all major operating systems but is not installed by default. For details, see [Install PowerShell](https://learn.microsoft.com/powershell/scripting/install/installing-powershell). Once PowerShell has been installed, run the following:
+
 ```powershell
+Install-Module -Name Az.Accounts
+Install-Module -Name Az.Resources
 Import-Module -Name FinOpsToolkit
```

Comment:
Should be Install-Module instead of Import-Module. Importing a module without installing it raises an error:

> Import-Module: The specified module 'FinOpsToolkit' was not loaded because no valid module file was found in any module directory.

Reply:
```suggestion
Install-Module -Name FinOpsToolkit
```